sys-cluster/ceph/ceph-18.2.4-r4.ebuild

```diff
@@ -414,6 +414,7 @@ src_configure() {
 }
 
 src_compile() {
+	export CMAKE_BUILD_PARALLEL_LEVEL=$(makeopts_jobs)
```

Comment:
We should consider setting this in the eclass. Do we have a bug for that? If not, could you file one?